core/arch/arm/plat-imx/conf.mk

```diff
@@ -1,29 +1,20 @@
 include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
 
-CROSS_COMPILE	?= arm-linux-gnueabihf-
-COMPILER	?= gcc
-
-CFG_ARM32_core ?= y
-CFG_MMU_V7_TTB ?= y
-
-core-platform-cppflags	 = -I$(arch-dir)/include
+core-platform-cppflags  = -I$(arch-dir)/include
 core-platform-subdirs += \
 	$(addprefix $(arch-dir)/, kernel mm tee sta) $(platform-dir)
 core-platform-subdirs += $(arch-dir)/sm
 
-libutil_with_isoc := y
-CFG_GENERIC_BOOT ?= y
-CFG_IMX_UART ?= y
-CFG_MMU_V7_TTB ?= y
-CFG_NO_TA_HASH_SIGN ?= y
-CFG_PM_STUBS ?= y
-CFG_SECURE_TIME_SOURCE_CNTPCT := y
-CFG_WITH_SOFTWARE_PRNG ?= y
-CFG_WITH_STACK_CANARIES := y
+$(call force,libutil_with_isoc,y)
+$(call force,CFG_ARM32_core,y)
```

Comment:
`$(call force,CFG_ARM64_core,n)` each time we have `$(call force,CFG_ARM32_core,y)`